### PR TITLE
[BitwiseCopyable] Promote to feature.

### PR DESCRIPTION
**Pull request by nate-chandler**
SE-0426 was accepted.

### CODE REVIEW

**Comment by nate-chandler** [comment]
@swift-ci please test

**Comment by nate-chandler** [comment]
@swift-ci please test

**Comment by nate-chandler** [comment]
@swift-ci please test

**Comment by nate-chandler** [comment]
@swift-ci please test source compatibility

**Comment by nate-chandler** [comment]
@swift-ci please apple silicon benchmark

**Comment by nate-chandler** [comment]
@swift-ci please test

**Comment by nate-chandler** [comment]
@swift-ci please test source compatibility

File: include/swift/Basic/Features.def [hunk 1]
```diff
@@ -179,6 +179,7 @@ LANGUAGE_FEATURE(BuiltinAddressOfRawLayout, 0, "Builtin.addressOfRawLayout")
 LANGUAGE_FEATURE(MoveOnlyPartialConsumption, 429, "Partial consumption of noncopyable values")
 /// Enable bitwise-copyable feature.
 LANGUAGE_FEATURE(BitwiseCopyable, 426, "BitwiseCopyable protocol")
+SUPPRESSIBLE_LANGUAGE_FEATURE(ConformanceSuppression, 426, "Suppressible inferred conformances")
```
Comment (atrick) [inline include/swift/Basic/Features.def:182]: Is ConformanceSuppression considered part of 426 or 428?

**Review by atrick — APPROVED** [review]
Great!

File: include/swift/Basic/Features.def (hunk 1 above)
Comment (nate-chandler) [reply include/swift/Basic/Features.def:182]: It's part of 426: [06921cfe84d3a5025d296e0a462db8a9ad8ac413](https://github.com/apple/swift-evolution/blob/main/proposals/0426-bitwise-copyable.md#suppressing-inferred-conformance)

**Pull request merged**